docbook/src/wadl/autoscale.wadl

```diff
@@ -318,8 +316,8 @@
             <param style="plain" name="args" required="true"
                 type="object" path="$.launchConfiguration.args">
                 <wadl:doc xmlns="http://docbook.org/ns/docbook"
-                    xml:lang="EN"><para>The item to be configured.
-                        Must be <parameter>server</parameter> or <parameter>loadBalancers</parameter>. Most
+                    xml:lang="EN"><para>The configuration used to create new servers in the scaling group.
+                        You must specify <parameter>server</parameter> and/or <parameter>loadBalancers</parameter> as attributes. Most
```

Comment:
This shouldn't be "and/or": you have to specify `server`, but load balancers are optional. However, you can just not specify that at all, because the documentation for `server` and `loadBalancers` itself already correctly specifies them as mandatory and optional respectively.